packages/react/src/components/Toggle/Toggle.tsx

```diff
@@ -129,80 +129,80 @@ export function Toggle({
   );
 }
 
-Toggle.propTypes = {
+type ToggleProps = {
```

Comment:
We can't remove propTypes. Instead we need to copy the proptypes def to above the component definition and then retool the proptypes to be a ts interface.

Please read the documentation on the [steps to provide baseline type definitions for components](https://github.com/carbon-design-system/carbon/blob/main/docs/guides/adding-component-types.md#steps-to-provide-baseline-type-definitions-for-components) - this and more is outlined there.